karrot/utils/email_utils.py

```diff
@@ -96,6 +97,12 @@ def prepare_email(
         raise Exception('email context should not redefine defaults: ' + ', '.join(redefined_keys))
     context.update(default_context)
 
+    if 'header_image' not in context:
+        if 'group' in context and context['group'].photo:
+            context['header_image'] = context['group'].photo.url
+        else:
+            context['header_image'] = settings.DEFAULT_EMAIL_HEADER_IMAGE
```

Comment:
This seems to result in `{HOSTNAME}https://user-images.githubusercontent.com/31616/36565633-517373a4-1821-11e8-9948-5bf6887c667e.png`, at least I didn't notice anything that should prevent it.

Reply:
Ah yeah, of course. Doh!

Reply:
Should be fixed now. I'm trying to learn how to program again...